cellmlmanip/rdf.py

```diff
@@ -63,10 +63,20 @@ def create_rdf_node(self, node_content=None, fragment_id=None):
 '''
 
 
-def create_rdf_node(namespace_uri, local_name):
+def create_rdf_node(namespace_uri, local_name=None):
```

Comment:
This is horrible! It should take a single argument always, since that's what it's always called with now. i.e. `def create_rdf_node(node_content):`

The docs also need updating.

Then the body can be like
```python
if isinstance(node_content, str):
    return rdflib.Literal(node_content)
else:
    namespace_uri, local_name = node_content
    # continue as originally
```

Reply:
hhm good point. I was struggeling to keep the same method signature but maybe that's not possible